compiler/rustc_lexer/src/cursor.rs

```diff
@@ -61,8 +62,8 @@ impl<'a> Cursor<'a> {
     }
 
     /// Returns amount of already consumed symbols.
-    pub(crate) fn len_consumed(&self) -> usize {
-        self.initial_len - self.chars.as_str().len()
+    pub(crate) fn len_consumed(&self) -> u32 {
+        (self.initial_len - self.chars.as_str().len()).try_into().unwrap()
```

Comment:
Would this be clearer?
```suggestion
        let len = self.initial_len - self.chars.as_str().len();
        len.try_into().unwrap()
```
I wonder do we just unwrap if the integer overflow?

Reply:
Changed. 
Yes, it panics on overflow. It was suggested by the compiler and I guess it's more explicit than `as u32`.